rllib/agents/pg/tests/test_pg.py

```diff
@@ -26,10 +29,36 @@ def test_pg_compilation(self):
         config = pg.DEFAULT_CONFIG.copy()
         config["num_workers"] = 1
         config["rollout_fragment_length"] = 500
+        # Test with filter to see whether they work w/o preprocessing.
+        config["observation_filter"] = "MeanStdFilter"
         num_iterations = 1
 
+        image_space = Box(-1.0, 1.0, shape=(84, 84, 3))
+        simple_space = Box(-1.0, 1.0, shape=(3, ))
+
+        tune.register_env(
```

Comment:
We don't for envs. The only registry we have in RLlib is for RLlib ModelV2s:

```
from ray.rllib.models.catalog import ModelCatalog

ModelCatalog.register_custom_model("my_model", MyModelClass)
```

Reply:
ah ok. that feels a bit weird. 👌

Reply:
Yeah, I'm not sure these registries are very useful. It can afaik all be done via config settings, like:

```
env: RandomEnv
env_config:
    key1: ..
    key2: ..
```